test/e2e/generator/generate.go

```diff
@@ -184,7 +184,7 @@ func generateTestnet(r *rand.Rand, opt map[string]interface{}) (e2e.Manifest, er
 		numValidators = 4
 	case "large":
 		// FIXME Networks are kept small since large ones use too much CPU.
-		numSeeds = r.Intn(1)
+		numSeeds = r.Intn(2)
```

Comment:
Left over from something else?

Reply:
I made this change. Some static analysis tool picked it up. I believe `r.Intn(1)` is always 0